scripts/tasks.sh

```diff
@@ -490,32 +490,32 @@ spec:
   template:
     spec:
       tolerations:
-        - key: node-role.kubernetes.io/master
+        - key: node-role.kubernetes.io/control-plane
           operator: Exists
       containers:
         - name: rook-ceph-operator
           env:
             - name: DISCOVER_TOLERATION_KEY
-              value: node-role.kubernetes.io/master
+              value: node-role.kubernetes.io/control-plane
             - name: CSI_PROVISIONER_TOLERATIONS
               value: |
-                - key: node-role.kubernetes.io/master
+                - key: node-role.kubernetes.io/control-plane
```

Comment:
Rather than remove the existing `node-role.kubernetes.io/master` toleration can we append `node-role.kubernetes.io/control-plane` as an additional toleration?